src/murmur/Server.cpp

```diff
@@ -1197,8 +1197,21 @@ void Server::newClient() {
 
 		sock->setPrivateKey(qskKey);
 		sock->setLocalCertificate(qscCert);
+
+		// Treat the leaf certificate as a root.
+		// This shouldn't strictly be necessary,
+		// and is a left-over from early on.
+		// Perhaps it is necessary for self-signed
+		// certs?
 		sock->addCaCertificate(qscCert);
-		sock->addCaCertificates(qlCA);
+
+		// Add CA certificates specified via
+		// murmur.ini's sslCA option.
+		sock->addCaCertificates(Meta::mp.qlCA);
+
+		// Add intermediate CAs found in the PEM
+		// bundle used for this server's certificate.
+		sock->addCaCertificates(qlIntermediates);
```

Comment:
> I’m a little confused why we add these.
> We can't really verify client certs against our server cert signers!?
> The user certs will be generated and signed by others.

Yes, it is true that it seems a little stretched to add the intermediates -- but that was what happened before this PR as well -- the intermediates were added to `QSslSocket::defaultCaCertificates()` here: https://github.com/mumble-voip/mumble/pull/2860/commits/273ee9516fd222eec07e095e7debb99c875a7644#diff-eb4bf8c73b2b6962f58d9d5e0a3b79dcL431